lms/djangoapps/ccx/migrations/0003_auto__del_ccxfuturemembership__del_ccxmembership.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Deleting model 'CcxFutureMembership'
+        db.delete_table('ccx_ccxfuturemembership')
+
+        # Deleting model 'CcxMembership'
+        db.delete_table('ccx_ccxmembership')
+
+
+    def backwards(self, orm):
```

Comment:
Can we leave this migration out for now and make the rollback for the previous migration a noop? That way we can have an easier rollback in case something goes wrong?